policies/releasestrat.html

```diff
@@ -167,9 +167,9 @@ <h5>
 	    <li>No existing public interface can be modified except where changes
 	    are unlikely to break source compatibility.
 	    <li>No existing public interface can be removed until its replacement
-	    has been in place in an LTS stable release and the original function has
-	    been documented as deprecated for at least 5 years. A public interface
-	    is any function or macro declared in a public header file.</li>
+	    has been in place in an LTS stable release. The original function must
+	    also have been documented as deprecated for at least 5 years. A public
+	    interface is any function or macro declared in a public header file.</li>
```

Comment:
I think we need to say "structure" as well.

Reply:
Oh, and just wanted to say that I like the rephrase you ended up doing.  Thank you.

Reply:
I added "structure" as well. I also changed "The original function" to say "The original interface".